dem_handler/utils/spatial.py

```diff
@@ -439,14 +407,14 @@ def resize_bounds(bounds: BoundingBox, scale_factor: float = 1.0) -> BoundingBox
     BoundingBox
         Resized bounding box.
     """
-    x_dim = bounds.xmax - bounds.xmin
-    y_dim = bounds.ymax - bounds.ymin
+    x_dim = bounds.right - bounds.left
+    y_dim = bounds.top - bounds.bottom
 
     dx = ((scale_factor - 1) * x_dim) / 2
     dy = ((scale_factor - 1) * y_dim) / 2
 
     return BoundingBox(
-        bounds.xmin - dx, bounds.ymin - dy, bounds.xmax + dx, bounds.ymax + dy
+        bounds.left - dx, bounds.bottom - dy, bounds.right + dx, bounds.top + dy
```

Comment:
Do we need to add a check/step to enforce that the new bounds don't go beyond the standard extents? e.g. bounds.bottom - dy should never be less than -90?